_data/simple-icons.json

```diff
@@ -1669,8 +1669,11 @@
             "title": "DHL",
             "hex": "FFCC00",
             "source": "https://www.dpdhl-brands.com/dhl/en/guides/design-basics/logo-and-claim.html"
-        },
-        {
+        }, {
+            "title": "Diagrams.net",
```

Comment:
Sorry for not catching it before. Looks like they always use a lowercase `d` for their brand name? What do you think?

Reply:
Yep - absolutely right! Pushed update just now.